src/components/CurratedContentCard/index.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { Image } from '../Image'
+import {
+  color,
+  radius,
+  space,
+  fontSize,
+  fontWeight,
+  device,
+  lineHeight,
+} from '../../theme'
+import { SmallText } from '../Text'
+import { transition } from '../../../src'
+
+const Container = styled.div`
+  background-color: ${color.elevatedBackground};
+  display: flex;
+  inline-size: 302px;
+  block-size: 314px;
+  flex-direction: column;
+  cursor: pointer;
+  border-radius: ${radius.lg};
+  overflow: hidden;
+
+  @media ${device.tablet} {
+    inline-size: 639px;
+    block-size: 244px;
+    flex-direction: row;
+  }
+
+  @media ${device.laptop} {
+    &:hover {
+      img {
+        transition: all ${transition};
+        transform: scale(1.1);
+      }
+    }
+    &:not(:hover) {
+      img {
+        transition: all ${transition};
+        transform: scale(1);
+      }
+    }
+  }
+`
+const Content = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  padding: ${space[16]};
+  max-block-size: 127px;
+  max-inline-size: 302px;
+
+  @media ${device.tablet} {
+    max-inline-size: 244px;
+    max-block-size: 244px;
+    padding-inline: ${space[24]};
+    padding-block: ${space[40]};
+  }
+`
+const ImageHolder = styled.div`
+  min-inline-size: 302px;
+  overflow: hidden;
+  border-start-start-radius: ${radius.lg};
+  border-start-end-radius: ${radius.lg};
+
+  @media ${device.tablet} {
+    min-inline-size: 395px;
+    border-start-end-radius: 0;
+    border-end-start-radius: ${radius.lg};
+  }
+`
+const Title = styled.h4`
+  font-weight: ${fontWeight.semiBold};
+  font-size: ${fontSize[16]};
+  line-height: ${lineHeight.copy};
+  padding-block-end: ${space[4]};
+
+  @media ${device.tablet} {
+    font-size: ${fontSize[18]};
+    padding-block-end: ${space[12]};
+  }
+`
+const Body = styled(SmallText)`
+  color: ${color.foregroundMuted};
+  overflow: hidden;
+
+  @media ${device.tablet} {
+    font-size: ${fontSize[16]};
+  }
+`
+
+interface Props {
+  imageUrl: string
+  title: string
+  text?: string
+}
+
+export const CurratedContentCard = ({ imageUrl, title, text }: Props) => {
```

Comment:
I think `CurratedContentCard` is not a good name for something like this. It can be used with currated content, and maybe for TicketSwap it's only for currated content, but looking broader at this it can be used for any content. Maybe something like `ContentCard` or `ImageCard` is a better name?

Also saw that we use this for the Carousel, so if it's for that, I think it shouldn't live on it's own, but it should be located inside the Carousel folder.